README.md

```diff
@@ -67,3 +67,15 @@ $ terminus help workflows
 
 The `list` and `show` commands will allow you to explore previous workflows and their Quicksilver operations. The `watch` command is a developers best friend: it will set up Terminus to automatically "follow" the workflow activity of your site, dumping back any Quicksilver output along with them.
 
+## Troubleshooting ##
+- `pantheon.yml` goes in your drupal root, ie. your ~/code directory
+- `private` also goes in your drupal root, and is git-ignored by default. You must `--force` add any and all changes
```

Comment:
> is git-ignored by default

`~code/private` should not be gitignored by default. That's the explicit place on the platform where we document customers commit things they do not want exposed on the web. It's a bug with us upstream that this is listed in our Drupal 8 stock `.gitignore` file and we should deal with it as such.

No `--force` pushing should be necessary to get Quicksilver working.